rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
To be clear, this is the summary on evaluation duration. We have separate total/error counters.